src/automation.md

```diff
@@ -0,0 +1,107 @@
+# Automation Process
+Level: *Skilled Gorilla*
+
+## Preparation
+
+Developer and operations are no longer the exclusive responsibility of DevOps engineers alone. Today, Frontend Developers help to reshape the form and the way process occurs within their projects improving collaboration between Frontends, Backends, and Stakeholders itself. <br/>
```

Comment:
You create a line break in Markdown by adding double spaces at the end of the line, instead of `<br/>`.

Example:
This  
sentence  
should  
be  
broke  
into  
separate  
words  
per  
line.

Source:
```md
This  
sentence  
should  
be  
broke  
into  
separate  
words  
per  
line.
```

Reply:
Sorry, didn't get it

Reply:
I was just saying that in Markdown double spaces (`  `) will be transformed into `<br/>`.
If you select the "source" text you will notice the extra spaces at the end of every line.